### ISSUE TITLE
Annotated Bibliography provides no information about how to modify it

### ISSUE THREAD

**Issue by tgraham-antenna** [markupdeclaration/markupdeclaration.org#15]
The Annotated Bibliography invites additional annotations, but provides no information about how to do that.

The link between https://markupdeclaration.org/resources/bibliography and `markupdeclaration.org/_pages/DeclarativeMarkupDocs.html` isn't obvious from outside the source HTML.

Also, the second paragraph has 'markkup'.

**Comment by ndw** [comment]
Indeed. The downside of making the site self publishing in the easiest possible way is that the content is a bit ... tangled in the repo. Will think about how to fix that.
